mne/viz/_scraper.py

```diff
@@ -0,0 +1,36 @@
+# Authors: Eric Larson <larson.eric.d@gmail.com>
+#
+# License: Simplified BSD
+
+import time
+
+
+class _PyQtGraphScraper:
+
+    def __repr__(self):
+        return '<PyQtGraphScraper>'
+
+    def __call__(self, block, block_vars, gallery_conf):
+        from mne_qt_browser._pg_figure import PyQtGraphBrowser
+        from sphinx_gallery.scrapers import figure_rst
+        from PyQt5.QtWidgets import QApplication
+        for gui in block_vars['example_globals'].values():
+            if (isinstance(gui, PyQtGraphBrowser) and
+                    not getattr(gui, '_scraped', False) and
+                    gallery_conf['builder_name'] == 'html'):
+                gui._scraped = True  # monkey-patch but it's easy enough
+                img_fname = next(block_vars['image_path_iterator'])
+                inst = QApplication.instance()
+                assert inst is not None
+            if (hasattr(self, 'load_thread')
+                    and self.load_thread is not None):
+                if self.load_thread.isRunning():
+                    gui.load_thread.wait(30000)
+                pixmap = gui.grab()
+                pixmap.save(img_fname)
+                gui.close()
+                inst.processEvents()
+                inst.processEvents()
```

Comment:
Is this still necessary?
```suggestion
```

Reply:
No idea, when I was testing locally before I think it was. Locally you can test in the `doc/` directory with `PATTERN=10_overview make html_dev-pattern` and it will rebuild just the one example for you.

Hopefully what I just pushed will at least give the right result, then we can try removing these and let CircleCI tell us if they were necessary

Reply:
I would like to test but for some reason I got now this error and I don't understand what's the problem.
```python
Configuration error:
There is a programmable error in your configuration file:

Traceback (most recent call last):
  File "C:\Users\marti\anaconda3\envs\mnedev\lib\site-packages\sphinx\config.p
y", line 328, in eval_config_file
    exec(code, namespace)
  File "C:\Users\marti\PycharmProjects\mne-python\doc\conf.py", line 26, in <m
odule>
    from mne.tests.test_docstring_parameters import error_ignores
ModuleNotFoundError: No module named 'mne.tests'
```
Maybe you can help me?

Reply:
Do you have the dev version installed with setup.py develop or pip install -e ?

Reply:
Oh yes, I installed from the wrong folder. Thank you!